public/app/features/explore/TraceView/TraceViewContainer.tsx

```diff
@@ -50,40 +63,71 @@ export function TraceViewContainer(props: Props) {
     (state: StoreState) => state.explore.panes[props.exploreId]?.datasourceInstance ?? undefined
   );
   const datasourceType = datasource ? datasource?.type : 'unknown';
+  const [ref, { width }] = useMeasure<HTMLDivElement>();
+
+  const links = useMemo(() => {
+    if (!traceProp) {
+      return [];
+    }
+    return getTraceLinks(traceProp);
+  }, [traceProp]);
 
   if (!traceProp) {
     return null;
   }
 
   return (
-    <div className={style.container}>
-      {!config.featureToggles.newTraceViewHeader && (
-        <TracePageSearchBar
-          navigable={true}
-          searchValue={search}
-          setSearch={setSearch}
-          spanFindMatches={spanFindMatches}
-          searchBarSuffix={searchBarSuffix}
-          setSearchBarSuffix={setSearchBarSuffix}
-          focusedSpanIdForSearch={focusedSpanIdForSearch}
-          setFocusedSpanIdForSearch={setFocusedSpanIdForSearch}
-          datasourceType={datasourceType}
-        />
-      )}
-      <TraceView
-        exploreId={exploreId}
-        dataFrames={dataFrames}
-        splitOpenFn={splitOpenFn}
-        scrollElement={scrollElement}
-        traceProp={traceProp}
-        spanFindMatches={spanFindMatches}
-        search={search}
-        focusedSpanIdForSearch={focusedSpanIdForSearch}
-        queryResponse={queryResponse}
-        datasource={datasource}
-        topOfViewRef={topOfViewRef}
-        topOfViewRefType={TopOfViewRefType.Explore}
-      />
+    <div className={style.container} ref={ref}>
+      <PanelChrome
+        padding="none"
+        width={width}
+        title={getTraceName(traceProp.spans)}
+        titleItems={
+          <span className={style.duration}>
+            {config.featureToggles.newTraceViewHeader ? formatDuration(traceProp.duration) : traceProp.traceID}
+            {!config.featureToggles.newTraceViewHeader && links && links.length > 0 && <ExternalLinks links={links} />}
+          </span>
+        }
+        displayMode="transparent"
```

Comment:
there's a bunch of custom styles in `style.container` that are used to make `PanelChrome` look like it had `displayMode="default"`, we should remove them and use `displayMode="default"` instead